src/pathop.rs

```diff
@@ -15,6 +17,7 @@ impl PathOp {
             path: path.to_path_buf(),
             op: op,
             cookie: cookie,
+            time: SystemTime::now(),
```

Comment:
I don't think adding this field is useful. Could you explain why you added it?

A timestamp would perhaps make sense if it was reported by notify, one layer down. But even then, at the platform layer, there is no guarantee of time for any filesystem event, which is one reason I've been hesitant to add timestamps in the past. Such a timestamp would mean "when was this event received by [some subsystem of the library]" rather than the more intuitive "when was this event _caused_".

> to give additional information about the timestamp **the change was made.**

<sup>(emphasis mine)</sup>

(In inotify, the events as received by inotify and then as passed on to notify can usually be expected to be pretty close — sub-milliseconds close — to the actual system call that triggered them. In fsevent, the events may be 30 **seconds** or even more time away from the "actual write to the system" that caused them! And that's raw platform time, not counting any debouncing that might occur in userspace.)

If a consumer wants to associate a timestamp to an event, they probably should do that themselves, when _they_ receive the event.